lib/internal/test_runner/harness.js

```diff
@@ -18,7 +19,7 @@ const {
 const { exitCodes: { kGenericUserError } } = internalBinding('errors');
 
 const { kEmptyObject } = require('internal/util');
-const { kCancelledByParent, Test, Suite } = require('internal/test_runner/test');
+const { kCancelledByParent, Test, TestHook, Suite } = require('internal/test_runner/test');
```

Comment:
Instead of importing `TestHook` and doing an `instanceof` check, we can check for `test.hookType`, which is all we really care about anyway.